### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by MarchCorbin**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Testing

## What is the current behavior?

- Testing it passing for the PokeDetails, Header tests and Game.js. Game is passing but seems to be having some conflict with Jest sixteen since the last merge. 

## What is the new behavior?

-
## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Components/Game/Game.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import Game from './Game'
+import { render, getByAltText } from '@testing-library/react'
+import { BrowserRouter } from 'react-router-dom'
+import '@testing-library/jest-dom'
+import { getAllPokemon, getSinglePokemon } from '../../ApiCalls'
+
+jest.mock('../../ApiCalls')
+
+describe('Game', () => {
+  it('should be true', () => {
```
Comment (attom2) [inline src/Components/Game/Game.test.js:12]: We can take this out since other tests are passing

File: src/Components/Header/Header.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import '@testing-library/jest-dom'
+
+describe('Header', () => {
+  it('should render header text', () => {
+    const { getByText } = render(
+      <BrowserRouter>
+      <Header />
+      </BrowserRouter>
+    )
+    const linkElement = getByText(/Who's That Pokémon?/i)
```
Comment (attom2) [inline src/Components/Header/Header.test.js:14]: We can call this something more semantic than linkElement, title?

File: src/Components/PokeDetails/PokeDetails.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PokeDetails from './PokeDetails'
+import '@testing-library/jest-dom'
+import { render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import Pokedex from '../Pokedex/Pokedex'
```
Comment (emontealvo) [inline src/Components/PokeDetails/PokeDetails.test.js:6]: Doesn't look like this variable is used.

File: src/Components/Game/Game.test.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        }, {
+          "name": "charmander",
+          "url": "https://pokeapi.co/api/v2/pokemon/4/"
+        }
+      ]}
+      get4RdmPokemon={ jest.fn() }
+      />
+      </BrowserRouter>
+    )
+    const gamePage = getByAltText("game-section")
```
Comment (emontealvo) [inline src/Components/Game/Game.test.js:39]: does this test still pass after Andy changed the props for Game?

**Pull request merged**